packages/webapp/pages/onboarding.tsx

```diff
@@ -348,7 +366,7 @@ export function OnboardPage(): ReactElement {
         className={classNames(
           'flex w-full flex-grow flex-wrap justify-center px-6 tablet:gap-10',
           !isFiltering && wrapperMaxWidth,
-          !isAuthenticating && 'mt-8 flex-1 content-center',
+          !isAuthenticating && 'mt-7.5 flex-1 content-center',
```

Comment:
Minor fix to avoid a majority overscroll issue.
(not 100% solved as certain sizes will have a small overlay, needed to have scroll when actually needed)